libcloud/httplib_ssl.py

```diff
@@ -187,17 +186,19 @@ def request(self, method, url, body=None, headers=None, raw=False):
             verify=self.ca_cert if self.ca_cert is not None else self.verify
         )
 
-    def prepared_request(self, method, url, body=None, headers=None, raw=False):
-        req = requests.Request(method, ''.join([self.host, url]), data=body, headers=headers)
-        
+    def prepared_request(self, method, url, body=None,
+                         headers=None, raw=False):
+        req = requests.Request(method, ''.join([self.host, url]),
+                               data=body, headers=headers)
+
         prepped = self.session.prepare_request(req)
 
         prepped.body = body
-        
-        self.response = self.session.send(prepped,
+
+        self.response = self.session.send(
+            prepped,
             stream=raw,
-            verify=self.ca_cert if self.ca_cert is not None else self.verify
-        )
+            verify=self.ca_cert if self.ca_cert is not None else self.verify)
```

Comment:
I've just checked in a fix to check if it's a list, if it is, then use the first one, if its bigger than 1, throw an error. People shouldn't need to use this anymore (that was the whole point) since requests comes with the Mozilla cert bundle pre-installed.